static/app/components/replays/replayController.tsx

```diff
@@ -15,117 +15,131 @@ import {formatTime} from './utils';
 
 const SECOND = 1000;
 
-interface ReplayControllerProps {
+interface Props {
```

Comment:
Getting rid of all the previous `*Props` types is great in here. 

No more needing to:
1. declare Public `ReplayControllerProps`
2. declare internal `ControlsProps` which is public + context
3. destructure vars from context value
4. pass destructured var in as props, mixed with public props

Now there are a few spots that do variable destructuring, but without the mountains of props.